### PR TITLE
DDBTEAM-527: Added limit to the allowed number of identifiers pr. req…

### PR DESCRIPTION
**Pull request by turegjorup**
…uest

### CODE REVIEW

File: src/Api/Dto/Cover.php [hunk 1]
```diff
@@ -49,10 +49,12 @@
  *                      {
  *                          "name" = "identifiers",
  *                          "in" = "query",
- *                          "description" = "A list of identifiers of {type}",
+ *                          "description" = "A list of identifiers of {type}. Maximum number os identifiers per reqeust is %d",
  *                          "required" = true,
  *                          "schema" : {
  *                              "type": "array",
+ *                              "maxLength": 200,
```
Comment (cableman) [inline src/Api/Dto/Cover.php:56]: Should 200 be substituted with `%d`?

File: src/Api/OpenApi/OpenApiDecorator.php [hunk 2]
```diff
@@ -44,6 +47,11 @@ public function normalize($object, $format = null, array $context = [])
         // "scopes" should be object, not array
         $docs['components']['securitySchemes']['oauth']['flows']['password']['scopes'] = new \stdClass();
 
+        // Set max identifier count from .env
+        $description = sprintf($docs['paths']['/api/v2/covers']['get']['parameters'][1]['description'], $this->maxIdentifierCount);
+        $docs['paths']['/api/v2/covers']['get']['parameters'][1]['description'] = $description;
+        $docs['paths']['/api/v2/covers']['get']['parameters'][1]['schema']['maxLength'] = 10;
```
Comment (cableman) [inline src/Api/OpenApi/OpenApiDecorator.php:53]: Should the 10 have been `$this->maxIdentifierCount`?

File: src/Api/Dto/Cover.php (hunk 1 above)
Comment (turegjorup) [reply src/Api/Dto/Cover.php:56]: Tried doing that. But then the annotation reader died. Have changed it to `"maxLength": "DECORATED_ENV_VALUE",`

File: src/Api/OpenApi/OpenApiDecorator.php (hunk 2 above)
Comment (turegjorup) [reply src/Api/OpenApi/OpenApiDecorator.php:53]: Fixed

**Pull request merged**